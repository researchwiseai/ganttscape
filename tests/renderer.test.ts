import { describe, it, expect } from 'vitest';
import { parseSchedule, renderSchedule } from '../src/index.js';
import { join } from 'path';

describe('renderSchedule', () => {
  it('renders simple schedule snapshot', () => {
    const fixture = join(__dirname, 'fixtures', 'simple-schedule.json');
    const schedule = parseSchedule(fixture);
    const output = renderSchedule(schedule);
    expect(output).toMatchSnapshot();
  });
});