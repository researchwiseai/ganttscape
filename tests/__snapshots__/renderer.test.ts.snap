// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renderSchedule > renders simple schedule snapshot 1`] = `
"       00:00:00 00:00:01 00:00:02 00:00:03
Task A ███░
  長い   ░███"
`;
